.ci/azure-pipelines-build.yml

```diff
@@ -1,110 +1,122 @@
 jobs:
   - job: Build
-    displayName: 'Build'
+    displayName: "Build"
 
     pool:
-      vmImage: 'ubuntu-latest'
+      vmImage: "ubuntu-latest"
 
     steps:
       - task: NodeTool@0
-        displayName: 'Install Node'
+        displayName: "Install Node"
         inputs:
-          versionSpec: '12.x'
+          versionSpec: "12.x"
```

Comment:
Should probably use same nodejs version as docker?
I don't know what the differences are between 12.x and 14, but I would expect one to have better performance / be better supported